gamestonk_terminal/portfolio/portfolio_controller.py

```diff
@@ -93,6 +85,7 @@ def print_help(self):
 [info]Graphs:[/info][cmds]
     rmr         graph your returns versus the market's returns
     al          displays the allocation of the portfolio
+    mdd         display portfolio drawdown[/cmds]
```

Comment:
Why is the name `mdd`?

Reply:
I think it's the first letters of "maximum draw down"

Reply:
> I think it's the first letters of "maximum draw down"

@DidierRLopes This. The common jargon